### DISCUSSION TITLE
イシューとディスカッションの使い分け

### DISCUSSION THREAD

**Discussion by motoi** [copilot-jp/project-sprint#377] [Development]
イシューとディスカッションの使い分けは以下が良さそうな予感がする。

- ディスカッションは、わちゃわちゃ話して意見を言い合う砂場
- イシューはディスカッションの結果をまとめてログとして後から見やすくする整理された場所

イシューがコメントだらけだと記録としてあとから読みづらくなった記憶が多々あるので。

**Comment by KokoroKagawa** [comment]
↑同意です。
まずはディスカッションで意見を言い合って、タスクとして切り出せる（実際にorgに反映させる文章を考える等）段階になったらイシューにする、くらいの運用がいいかなと。

ちなみにディスカッションのイシュー化は画面右サイドのナビから可能です。
（ディスカッションの最初の投稿が下書きとしてベタ打ちされるので、その後の議論を整理してイシュー化すればOK。）

![create issue](https://user-images.githubusercontent.com/84454164/183863231-0b860633-6274-4c38-8f96-a4d3643b185c.png)

イシューをディスカッションにconvertすることも、同様にイシュー画面右サイドのナビから可能です。
![convert to discussion](https://user-images.githubusercontent.com/84454164/183863244-cfabe392-141e-42ef-a6e9-e8500e2dc16b.png)
